### PR TITLE
included the functionality for adding teams/projects, assign is there…

### PR DESCRIPTION
**Pull request by danieljtate**
… and needs a bit more work

### CODE REVIEW

File: app/components/Users.js [hunk 1]
```diff
@@ -162,6 +162,127 @@ var Users = React.createClass({
 
 var UsersPage = React.createClass({
 
+getTeam: function (){
```
Comment (AlexJYoungman) [inline app/components/Users.js]: Add space after brackets.
`function () {`

File: app/components/Users.js [hunk 2]
```diff
@@ -162,6 +162,127 @@ var Users = React.createClass({
...
+
+    var self = this;
+    axios.get(ServiceURL + 'project')
+    .then(response => {
+      self.setState({
+        projects: response.data
+      });
+    })
+    .catch(function (err) {
+
```
Comment (AlexJYoungman) [inline app/components/Users.js]: Remove unnecessary blank spaces here and line 179

File: app/components/Users.js [hunk 3]
```diff
@@ -162,6 +162,127 @@ var Users = React.createClass({
...
+      alert("Error 404: There is an issue when connecting to the web service.")
+    });
+},
+
+componentDidMount: function (){
```
Comment (AlexJYoungman) [inline app/components/Users.js]: Same again `function () {`

File: app/components/Users.js [hunk 4]
```diff
@@ -162,6 +162,127 @@ var Users = React.createClass({
...
+        });
+      }
+      catch (err) {
+        alert("Error 404: There is an issue when connecting to the web service.");
+        console.log(err);
+      }
+    },
+
+    createProject: function () {
+          var self = this;
```
Comment (AlexJYoungman) [inline app/components/Users.js]: Indentation™

File: app/components/Users.js [hunk 5]
```diff
@@ -162,6 +162,127 @@ var Users = React.createClass({
...
+              });
+            });
+          }
+          catch (err) {
+            alert("Error 404: There is an issue when connecting to the web service.");
+            console.log(err);
+          }
+        },
+
+        addUserToTeam: function () {
```
Comment (AlexJYoungman) [inline app/components/Users.js]: Whole class needs indentation™ sorting

File: app/components/Users.js [hunk 6]
```diff
@@ -162,6 +162,127 @@ var Users = React.createClass({
...
+                     .then(data => {
+                       self.setState({
+                         user: data.data
+                       });
+                     });
+                   });
+                 }
+                 catch (err) {
+                   alert("Error 404: There is an issue when connecting to the web service.");
+                   console.log(err);
```
Comment (AlexJYoungman) [inline app/components/Users.js]: Remove all `console.log`s

File: app/components/Users.js [hunk 7]
```diff
@@ -200,55 +312,36 @@ var UsersPage = React.createClass({
...
+            </div>
             <div className="mCellsCount">
             </div>
           </div>
         </tr>
       );
-    },this)
-
-    var newProjects = [];
+    },)
```
Comment (AlexJYoungman) [inline app/components/Users.js]: Remove `,` and add `;` after bracket

File: app/styles/css/users.css [hunk 8]
```diff
@@ -365,33 +373,39 @@ select {
...
 
 input[type=checkbox] {
   height:20px;
   width:20px;
-  position: absolute;
+  position:absolute;
   margin-top: 0;
   opacity:0;
+  margin-left: 0px;
+
```
Comment (AlexJYoungman) [inline app/styles/css/users.css]: Remove blank line

File: app/components/Users.js [hunk 9]
```diff
@@ -162,6 +162,125 @@ var Users = React.createClass({
 
 var UsersPage = React.createClass({
```
Comment (AlexJYoungman) [inline app/components/Users.js]: This class needs Indentation™

File: app/components/Users.js [hunk 10]
```diff
@@ -162,6 +162,125 @@ var Users = React.createClass({
...
+      self.setState({
+        teams: response.data
+      });
+    })
+    .catch(function (err) {
+      alert("Error 404: There is an issue when connecting to the web service.")
+    });
+},
+
+getProject: function (){
```
Comment (AlexJYoungman) [inline app/components/Users.js]: Space after () `function () {`

File: app/components/Users.js [hunk 11]
```diff
@@ -220,68 +218,68 @@ createTeam: function () {
...
+        });
+      }
+      catch (err) {
+        alert("Error 404: There is an issue when connecting to the web service.");
+        console.log(err);
+      }
+    },
+
+  addUserToTeam: function () {
+       var self = this;
```
Comment (TheGarrett) [inline app/components/Users.js]: indentation here is indented by 4 spaces, not 2

File: app/components/Users.js [hunk 12]
```diff
@@ -162,6 +162,125 @@ var Users = React.createClass({
 
 var UsersPage = React.createClass({
 
+getTeam: function () {
+    var self = this;
```
Comment (TheGarrett) [inline app/components/Users.js]: indentation here is indented by 4 spaces, not 2

File: app/components/Users.js [hunk 13]
```diff
@@ -162,6 +162,125 @@ var Users = React.createClass({
...
+    });
+},
+
+componentDidMount: function () {
+    this.getTeam();
+    this.getProject();
+},
+
+createTeam: function () {
+      var self = this;
```
Comment (TheGarrett) [inline app/components/Users.js]: indentation here is indented by 4 spaces, not 2

File: app/components/Users.js [hunk 14]
```diff
@@ -162,6 +162,125 @@ var Users = React.createClass({
...
+        projects: response.data
+      });
+    })
+    .catch(function (err) {
+      alert("Error 404: There is an issue when connecting to the web service.")
+    });
+},
+
+componentDidMount: function () {
+    this.getTeam();
```
Comment (TheGarrett) [inline app/components/Users.js]: indentation here is indented by 4 spaces, not 2

File: app/components/Users.js [hunk 15]
```diff
@@ -162,6 +162,125 @@ var Users = React.createClass({
...
+        teams: response.data
+      });
+    })
+    .catch(function (err) {
+      alert("Error 404: There is an issue when connecting to the web service.")
+    });
+},
+
+getProject: function (){
+    var self = this;
```
Comment (TheGarrett) [inline app/components/Users.js]: indentation here is indented by 4 spaces, not 2

File: app/components/Users.js [hunk 16]
```diff
@@ -162,6 +162,125 @@ var Users = React.createClass({
 
 var UsersPage = React.createClass({
 
+  getTeam: function () {
+      var self = this;
```
Comment (AlexJYoungman) [inline app/components/Users.js]: indentation™ here is indented by 4 spaces, not 2

File: app/components/Users.js [hunk 17]
```diff
@@ -162,6 +162,125 @@ var Users = React.createClass({
...
+          teams: response.data
+        });
+      })
+      .catch(function (err) {
+        alert("Error 404: There is an issue when connecting to the web service.")
+      });
+  },
+
+  getProject: function () {
+      var self = this;
```
Comment (AlexJYoungman) [inline app/components/Users.js]: indentation™ here is indented by 4 spaces, not 2

File: app/components/Users.js [hunk 18]
```diff
@@ -162,6 +162,125 @@ var Users = React.createClass({
...
+          projects: response.data
+        });
+      })
+      .catch(function (err) {
+        alert("Error 404: There is an issue when connecting to the web service.")
+      });
+  },
+
+  componentDidMount: function () {
+      this.getTeam();
```
Comment (AlexJYoungman) [inline app/components/Users.js]: indentation™ here is indented by 4 spaces, not 2

File: app/components/Users.js [hunk 19]
```diff
@@ -162,6 +162,125 @@ var Users = React.createClass({
...
+      this.getProject();
+  },
+
+  createTeam: function () {
+      var self = this;
```
Comment (AlexJYoungman) [inline app/components/Users.js]: indentation™ here is indented by 4 spaces, not 2

File: app/components/Users.js [hunk 20]
```diff
@@ -162,6 +162,125 @@ var Users = React.createClass({
...
+        });
+      }
+      catch (err) {
+        alert("Error 404: There is an issue when connecting to the web service.");
+        console.log(err);
+      }
+    },
+
+  createProject: function () {
+      var self = this;
```
Comment (AlexJYoungman) [inline app/components/Users.js]: indentation™ here is indented by 4 spaces, not 2

File: app/components/Users.js [hunk 21]
```diff
@@ -162,6 +162,125 @@ var Users = React.createClass({
...
+        });
+      }
+      catch (err) {
+        alert("Error 404: There is an issue when connecting to the web service.");
+        console.log(err);
+      }
+    },
+
+  addUserToTeam: function () {
+     var self = this;
```
Comment (AlexJYoungman) [inline app/components/Users.js]: indentation™ here is indented by 3 spaces, not 2

File: app/components/Users.js [hunk 22]
```diff
@@ -200,459 +309,429 @@ var UsersPage = React.createClass({
...
 
-    var filterModalProjects = newProjects.map(function (obj, idx) {
-      return (
-        //displays the list of projects within the filter modal
-        <tr className="table-users" key={idx}>
-          <div id="filterUserCheckbox">
-            <input type="checkbox" className="resetCheckbox" name="usersCheckbox" defaultValue="valuable" id="usersCheckbox" /><label htmlFor="usersCheckbox" />
+  return (
+    <div id="UsersContainer">
+    <Navbar />
```
Comment (TheGarrett) [inline app/components/Users.js]: this section

File: app/components/Users.js [hunk 23]
```diff
@@ -200,459 +309,429 @@ var UsersPage = React.createClass({
...
                 </div>
               </div>
             </div>
           </div>
         </div>
-        <Users ref='searchedUser'/>
       </div>
+      <Users ref='searchedUser'/>
+    </div>
     );
```
Comment (TheGarrett) [inline app/components/Users.js]: this section

**Pull request merged**